charts/delphai-keda/templates/deployment.yml

```diff
@@ -91,6 +91,24 @@ spec:
             - name: NVIDIA_DRIVER_CAPABILITIES
               value: "compute,utility"
             {{ end }}
+            {{ if .Values.mlflow }}
+            - name: MLFLOW_TRACKING_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: mlflow-username
+                  key: mlflow-username
+            - name: MLFLOW_TRACKING_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: mlflow-password
+                  key: mlflow-password
+            - name: DATABASE_URL
+              valueFrom:
+                secretKeyRef:
+                  name: mlflow-store-connection-string
+                  key: mlflow-store-connection-string
```

Comment:
Did you terraform this secrets? To all clusters?

Also it's maybe better to have one secret with 3 values, not a secret per values as they always used together

Reply:
They have been terraformed to staging and review. I happened to mention that it's better to have one secret that encompasses all 3, but it needs to be changed in Terraform

Reply:
Let's maybe have them in one secret?
https://atomleap.atlassian.net/browse/DEVOPS-267